docs/nim-cmds/web.md

```diff
@@ -0,0 +1,143 @@
+* [`nim web update WEBCONTENTPATH [NAMESPACE]`](#nim-web-update-webcontentpath-namespace)
+
+## `nim web clean [NAMESPACE]`
+
+Deletes all Content from Web Storage
+
+```
+USAGE
+  $ nim web clean [NAMESPACE]
+
+ARGUMENTS
+  NAMESPACE  The namespace to delete web content from (current namespace if omitted)
+
+OPTIONS
+  -f, --force        Just do it, omitting confirmatory prompt
+  -v, --verbose      Greater detail in error messages
+  --apihost=apihost  API host of the namespace to delete content from
+  --help             Show help
+```
+
+_See code: [src/commands/web/clean.ts](https://github.com//nimbella/nimbella-cli/blob/v1.9.3/src/commands/web/clean.ts)_
+
+## `nim web create WEBCONTENTPATH [NAMESPACE]`
+
+Adds Content to the Web Storage
+
+```
+USAGE
+  $ nim web create WEBCONTENTPATH [NAMESPACE]
+
+ARGUMENTS
+  WEBCONTENTPATH  Path to the content to be added
+  NAMESPACE       The namespace to add content to (current namespace if omitted)
+
+OPTIONS
+  -c, --cache=cache              Maximum amount of time in seconds, the web content is considered fresh, relative to the time of the request
+  -d, --destination=destination  Target location in web storage
+  -v, --verbose                  Greater detail in error messages
+  --apihost=apihost              API host of the namespace to add content to
+  --help                         Show help
+
+ALIASES
+  $ nim web add
+```
+
+_See code: [src/commands/web/create.ts](https://github.com//nimbella/nimbella-cli/blob/v1.9.3/src/commands/web/create.ts)_
+
+## `nim web delete WEBCONTENTNAME [NAMESPACE]`
+
+Deletes Content from the Web Storage
+
+```
+USAGE
+  $ nim web delete WEBCONTENTNAME [NAMESPACE]
+
+ARGUMENTS
+  WEBCONTENTNAME  The web content to be deleted
+  NAMESPACE       The namespace to delete content from (current namespace if omitted)
+
+OPTIONS
+  -v, --verbose      Greater detail in error messages
+  --apihost=apihost  API host of the namespace to delete web content from
+  --help             Show help
+```
+
+_See code: [src/commands/web/delete.ts](https://github.com//nimbella/nimbella-cli/blob/v1.9.3/src/commands/web/delete.ts)_
+
+## `nim web get WEBCONTENTNAME DESTINATION [NAMESPACE]`
+
+Gets Content from the Web Storage
+
+```
+USAGE
+  $ nim web get WEBCONTENTNAME DESTINATION [NAMESPACE]
+
+ARGUMENTS
+  WEBCONTENTNAME  The web content to get
+  DESTINATION     [default: ./] The location to write at
+  NAMESPACE       The namespace to get content from (current namespace if omitted)
+
+OPTIONS
+  -r, --url          Get web content url
+  -s, --save         Saves content on file system
+  -v, --verbose      Greater detail in error messages
+  --apihost=apihost  API host of the namespace to get web content from
+  --help             Show help
+  --saveAs=saveAs    Saves content on file system with the given name
+```
+
+_See code: [src/commands/web/get.ts](https://github.com//nimbella/nimbella-cli/blob/v1.9.3/src/commands/web/get.ts)_
```

Comment:
Nit: Are these links auto generated? There is an extra `/` in the URL. It looks like github parses the URI correctly so it's harmless, but would be nice to correct. The double `//` occur through out.

Reply:
The links are autogenerated by an oclif tool but I do have a post-massage step and can probably fix them there.   The post-massage step operates at a low textual level, which means this could be a slight pain to fix that way.   I am thinking that, down the road, I may stop using that particular oclif tool and instead manage this more like the way I manage the oclif-to-kui conversion for the workbench:  ask oclif to generate just a manifest and interpret that in my own tool.   The main inhibitor is that the current oclif tool formats all the help diagrams nicely for me.  So, I'll need to clone and own that bit of code.